internal/appsec/waf/waf_disabled.go

```diff
@@ -55,6 +53,13 @@ func (*Context) Run(map[string]interface{}, time.Duration) ([]byte, error) {
 	return nil, errDisabledReason
 }
 
+// TotalRuntime returns the cumulated waf runtime across various run calls within the same WAF context.
+// Returned time is in nanoseconds.
+func (*Context) TotalRuntime() uint64 { return uint64(0) }
```

Comment:
Literal numbers in Go are untyped and don't need conversions: 
```suggestion
func (*Context) TotalRuntime() uint64 { return 0 }
```